examples/tabs.rs

```diff
@@ -31,6 +31,45 @@ enum SelectedTab {
     Tab3,
 }
 
+impl Widget for SelectedTab {
```

Comment:
Move this below the Widget impl for App - to make this read top to bottom more easily.